ci-operator/step-registry/ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+	"path": "ipi/conf/vsphere/windows-machineset/ipi-conf-vsphere-windows-machineset-ref.yaml",
+	"owners": {
+		"approvers": [
+			"jhou1"
+		],
+		"reviewers": [
+			"jhou1",
+			"sunzhaohua2",
+			"huali9",
+			"miyadav"
```

Comment:
Could you add @rrasouli as reviewer please?

Reply:
done